kano_updater/scenarios.py

```diff
@@ -642,4 +642,7 @@ def beta_361_to_beta_370(self):
         pass
 
     def beta_370_to_beta_380(self):
-        pass
+        # linux kernel 4.4.21 shipped with Kano 3.8.0 emits systemd boot messages.
+        # fix by telling the kernel to enable an empty splash screen.
+        command="sed -i 's/systemd.show_status=0/splash/' {}".format('/boot/cmdline.txt')
```

Comment:
Unlikely to be necessary, but it's often useful to put `\b` at the start and end of teh pattern.